app-backend/app/src/test/scala/image/ImageSpec.scala

```diff
@@ -106,35 +106,35 @@ class ImageSpec extends WordSpec
             newImageDatasource1.toJson.toString()
           )
         ) ~> baseRoutes ~> check {
-          responseAs[Image]
+          val response = responseAs[Image.WithRelated]
```

Comment:
I'm half-surprised this didn't change the return type. I guess assignment returns the left-hand-side after assigning?